states/NV/governor.yaml

```diff
@@ -0,0 +1,171 @@
+contact_form:
+  steps:
+    # Note this is an iframe within http://gov.nv.gov/Contact/Email_the_Governor/
+    - visit: "https://nevada-voice.symplicity.com//contact_form/dee804819c38673f2500f9f724ac487b"
+    - select:
+        - name: "dnf_class_values[communication][primary_constituent][0][prefix]"
+          selector: "#dnf_class_values_communication__primary_constituent__0__prefix_"
+          value: $NAME_PREFIX
+          required: true
+          options:
+            "Miss": "Miss"
+            "Mr.": "Mr."
+            "Mrs.": "Mrs."
+            "Ms.": "Ms."
+            "Mr. and Mrs.": "Mr. and Mrs."
+            "Reverend": "Reverend"
+            "Father": "Father"
+            "Dr.": "Dr."
+            "Atty": "Atty"
+            "Prof": "Prof"
+            "Coach": "Coach"
+            "Ofc": "Ofc"
+    - fill_in:
+        - name: "dnf_class_values[communication][primary_constituent][0][fname]"
+          selector: "#dnf_class_values_communication__primary_constituent__0__fname_"
+          value: $NAME_FIRST
+          required: true
+        - name: dnf_class_values[communication][primary_constituent][0][lname]
+          selector: "#dnf_class_values_communication__primary_constituent__0__lname_"
+          value: $NAME_LAST
+          required: true
+        - name: "dnf_class_values[communication][primary_constituent][0][email]"
+          selector: "#dnf_class_values_communication__primary_constituent__0__email_"
+          value: $EMAIL
+          required: true
+        # Phone is not required by this form, but it is included here in case it needs to be implemented in the future.
+        - name: "dnf_class_values[communication][primary_constituent][0][phone]"
+          selector: "#dnf_class_values_communication__primary_constituent__0__phone_"
+          value: ""
+        - name: "dnf_class_values[communication][primary_constituent][0][address][0][street]"
+          selector: "#dnf_class_values_communication__primary_constituent__0__address__0__street_"
+          value: $ADDRESS_STREET
+        - name: "dnf_class_values[communication][primary_constituent][0][address][0][zip]"
+          selector: "#dnf_class_values_communication__primary_constituent__0__address__0__zip_"
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: "dnf_class_values[communication][primary_constituent][0][address][0][city]"
+          selector: "#dnf_class_values_communication__primary_constituent__0__address__0__city_"
+          value: $ADDRESS_CITY
+    - select:
+        - name: "dnf_class_values[communication][primary_constituent][0][address][0][state]"
+          selector: "#dnf_class_values_communication__primary_constituent__0__address__0__state_"
+          value: $ADDRESS_STATE_POSTAL_ABBREV
+          options:
+            "": ""
```

Comment:
Can probably drop this one, an empty string isn't too useful.

Reply:
Indeed.  I'll update it.